src/wabt.cpp

```diff
@@ -520,21 +80,151 @@ ExecutionResult WabtEngine::execute(
 ) {
   HERA_DEBUG << "Executing with wabt...\n";
 
-  // This is the wasm state
+  // Set up the wabt Environment, which includes the Wasm store
+  // and the list of modules used for importing/exporting between modules
   interp::Environment env;
 
-  // Lets instantiate our state
+  // Set up interface to eei host functions
   ExecutionResult result;
-
   // FIXME: shouldn't have this loose pointer here, but needed for setWasmMemory
   WabtEthereumInterface* interface = new WabtEthereumInterface{context, state_code, msg, result, meterInterfaceGas};
```

Comment:
I thought I have left this comment here a week ago: this is leaking memory now.

@chfast this was the reason I didn't merge it.

Reply:
Yes, reported by ASan. I will fix it.